airbyte-webapp/src/views/Connection/ConnectionForm/components/NamespaceField.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useMemo } from "react";
+import { Field, FieldProps } from "formik";
+import { FormattedMessage, useIntl } from "react-intl";
+import styled from "styled-components";
+
+import { ControlLabels, DropDown, Input } from "components";
+import { ConnectionNamespaceDefinition } from "core/domain/connection";
+
+const NamespaceConfigurationLabel = styled(ControlLabels)`
+  flex: 3 0 0;
+  & a {
+    color: ${({ theme }) => theme.greyColor40};
+  }
+`;
+
+const NamespaceFormatLabel = styled(ControlLabels)`
+  margin-left: 21px;
+  flex: 5 0 0;
+`;
+
+const Row = styled.div`
+  display: flex;
+  margin-bottom: 26px;
+`;
+
+const NamespaceField: React.FC<{}> = ({}) => {
+  const formatMessage = useIntl().formatMessage;
+
+  const definitions = useMemo(
+    () => [
+      {
+        value: ConnectionNamespaceDefinition.Source,
```

Comment:
@ChristopheDuong should all the values in this drop down display every time? or are there cases where we filter them out  (or don't throw the dropdown). e.g. in the case where the destination doesn't support namespaces

Reply:
You're right, we don't handle this yet...

Can we merge this in and re-work the show/hide afterward?

if you set up a namespace for a destination that does not support it, then it will just ignore it for the moment. So it should not be a big deal...

Otherwise, we are starting to get more "feature flags" including this new "supportsNamespace":
 
![Screenshot 2021-06-16 at 11 34 44](https://user-images.githubusercontent.com/7718175/122195986-1e612a80-ce97-11eb-9021-d6e34d23e699.png)

So maybe we should refactor into a single "features" map where we can easily include what features are supported by each connector?